SU2_CFD/include/drivers/CDriver.hpp

```diff
@@ -508,95 +521,95 @@ class CDriver {
 
   /*!
    * \brief Get the x component of the force density at a vertex on a specified marker.
    * \param[in] iMarker - Marker identifier.
    * \param[in] iVertex - Vertex identifier.
    * \return x component of the force density at the vertex.
    */
-  su2double GetVertexForceDensityX(unsigned short iMarker, unsigned short iVertex);
+  passivedouble GetVertexForceDensityX(unsigned short iMarker, unsigned short iVertex);
 
   /*!
    * \brief Get the y component of the force density at a vertex on a specified marker.
    * \param[in] iMarker - Marker identifier.
    * \param[in] iVertex - Vertex identifier.
    * \return y component of the force density at the vertex.
    */
-  su2double GetVertexForceDensityY(unsigned short iMarker, unsigned short iVertex);
+  passivedouble GetVertexForceDensityY(unsigned short iMarker, unsigned short iVertex);
 
   /*!
    * \brief Get the z component of the force density at a vertex on a specified marker.
    * \param[in] iMarker - Marker identifier.
    * \param[in] iVertex - Vertex identifier.
    * \return z component of the force density at the vertex.
    */
-  su2double GetVertexForceDensityZ(unsigned short iMarker, unsigned short iVertex);
+  passivedouble GetVertexForceDensityZ(unsigned short iMarker, unsigned short iVertex);
 
   /*!
    * \brief Set the x coordinate of a vertex on a specified marker.
    * \param[in] iMarker - Marker identifier.
    * \param[in] iVertex - Vertex identifier.
    * \param[in] newPosX - New x coordinate of the vertex.
    */
-  void SetVertexCoordX(unsigned short iMarker, unsigned short iVertex, su2double newPosX);
+  void SetVertexCoordX(unsigned short iMarker, unsigned short iVertex, passivedouble newPosX);
 
   /*!
    * \brief Set the y coordinate of a vertex on a specified marker.
    * \param[in] iMarker - Marker identifier.
    * \param[in] iVertex - Vertex identifier.
    * \param[in] newPosY - New y coordinate of the vertex.
    */
-  void SetVertexCoordY(unsigned short iMarker, unsigned short iVertex, su2double newPosY);
+  void SetVertexCoordY(unsigned short iMarker, unsigned short iVertex, passivedouble newPosY);
 
   /*!
    * \brief Set the z coordinate of a vertex on a specified marker.
    * \param[in] iMarker - Marker identifier.
    * \param[in] iVertex - Vertex identifier.
    * \param[in] newPosZ - New z coordinate of the vertex.
    */
-  void SetVertexCoordZ(unsigned short iMarker, unsigned short iVertex, su2double newPosZ);
+  void SetVertexCoordZ(unsigned short iMarker, unsigned short iVertex, passivedouble newPosZ);
 
   /*!
    * \brief Set the VarCoord of a vertex on a specified marker.
    * \param[in] iMarker - Marker identifier.
    * \param[in] iVertex - Vertex identifier.
    * \return Norm of the VarCoord.
    */
-  su2double SetVertexVarCoord(unsigned short iMarker, unsigned short iVertex);
+  passivedouble SetVertexVarCoord(unsigned short iMarker, unsigned short iVertex);
```

Comment:
I like seeing passivedouble here and there :) no need to pay for what is not being used.